apps/aetx/src/aetx.erl

```diff
@@ -171,10 +172,35 @@ new(Callback, Tx) ->
 fee(#aetx{ cb = CB, tx = Tx }) ->
     CB:fee(Tx).
 
--spec gas(Tx :: tx()) -> Gas :: non_neg_integer().
-gas(#aetx{ type = Type, cb = CB, size = Size, tx = Tx }) when Type =/= channel_offchain_tx ->
-    aec_governance:tx_base_gas(Type) + Size * aec_governance:byte_gas() + CB:gas(Tx);
-gas(#aetx{ type = channel_offchain_tx }) ->
+%% In case 0 is returned, the tx will not be included in the micro block
+%% candidate by the mempool.
+-spec gas(Tx :: tx(), Height :: aec_blocks:height()) -> Gas :: non_neg_integer().
+gas(#aetx{type = Type, cb = CB, size = Size, tx = Tx }, Height) when
```

Comment:
This function matches on transaction type for oracle transactions, accounting for state TTL, then has catch-all clause (`Type ... when Type =/= channel_offchain_tx`). It means that if code has typo in atom of transaction type (or they get misaligned) then code does not do what it is intended to do. I see there is test coverage in `aeoracle_SUITE` - checking that.

Reply:
One thing I noticed is that I could also add `CB:gas(Tx)` to oracle txs (even if all of them return 0)